src/elements/content-sidebar/additional-tabs/AdditionalTabs.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tabs components
+ * @author Box
+ */
+
+import React, { PureComponent } from 'react';
+import AdditionalTab from './AdditionalTab';
+import AdditionalTabsLoading from './AdditionalTabsLoading';
+import './AdditionalTabs.scss';
+
+type Props = {
+    tabs: ?AdditionalSidebarTabs,
+};
+
+type State = {
+    isLoading: boolean,
+};
+
+class AdditionalTabs extends PureComponent<Props, State> {
+    loadedTabs: boolean;
```

Comment:
Consider following the `is` or `has` prefix for boolean types. As this stands, it currently sounds like an array of tabs.

`hasLoadedTabs` or something

Reply:
forgot to remove, that's not being used.